content/docs/how-to-guides/connectivity-issues.md

```diff
@@ -54,9 +54,9 @@ This option is expected to work with all libpq-based apps.
 
 If your application or service uses golang Postgres clients like `pgx` and `lib/pg` you can set `sslmode=verify-full,` which will cause `SNI` info to be sent. Most likely, this was not intentional but happened inadvertently due to the golang's TLS library API design.
 
-### D. Specify the project name in the password field
+### D. Specify the project ID in the password field
 
-You can specify the project name in the password field as a last resort. So instead of your actual password (let it be 'my-pass'), you can set the following string, consisting of the project name and the password:
+You can specify the project ID in the password field as a last resort. So instead of your actual password (let it be 'my-pass'), you can set the following string, consisting of the project ID and the password:
```

Comment:
```suggestion
You can specify the project ID in the password field as a last resort. So, instead of your actual password (for example, 'my-pass'), you can set the following string, consisting of the project ID and the password:
```